reasonml/data_structures/stack.re

```diff
@@ -0,0 +1,26 @@
+type stack('a) = list('a);
+
+let make = () => [];
+
+let push = (s: stack('a), e: 'a) => [e, ...s]
+
+let pop = (s: stack('a)) => {
+  switch s {
+    | [_, ...tl] => [...tl]
```

Comment:
I can't remember the syntax well, but wouldn't `| [_, ...tl] => tl` work?

Reply:
I'll try